R/bs-theme-preview.R

```diff
@@ -77,6 +88,25 @@ bs_themer_ui <- function(opts, vals, theme) {
         if (!is.null(desc)) div(class = "form-text small", desc)
       )
     }
+
+    switch_input <- function(input_class = NULL, ...) {
+      # https://getbootstrap.com/docs/5.3/forms/checks-radios/#switches
+      div(
+        class = "form-row form-group form-check form-switch",
+        tags$label(
+          class = "form-check-label",
+          tags$input(
+            class = "form-check-input",
+            class = input_class,
+            type = "checkbox",
+            role = "switch",
+            id = paste0("bsthemer-", id)
+          ),
+          lbl
+        )
+      )
+    }
```

Comment:
tbh, I'm not sure why we're not using `input_switch()` or other Shiny input constructors in the themer app.

I'm going to push back on this slightly and say that I'd rather keep this here since it's just for the themer, rather than give the impression that we've thought about it in any other context. At some point we'll might do more work with the themer to full support dark more or to handle Bootstrap updates and we can refactor it then when we're more in that mindspace.